IaC/AKS/README.md

```diff
@@ -512,18 +500,18 @@ az monitor log-analytics workspace create -g $Ngsa_Smoker_RG -l $Ngsa_Location -
 cd $REPO_ROOT/IaC/scripts
 
 # Run the shell script to install smoker instances with ACI
-./smokers.sh -n $Ngsa_Name -r $Ngsa_Smoker_RG -s $Ngsa_App_Endpoint
+./smokers.sh -n $Ngsa_Name -r $Ngsa_Smoker_RG -s $Ngsa_Https_App_Endpoint
 
 ```
 
-Alternatively, you can deploy the smokers to AKS as cronjobs.
+Alternatively, you can deploy the smokers inside the AKS cluster as cronjobs.
```

Comment:
webv should be deployed as a pod that runs constantly - deploying load tests as a job is fine, but the baseline should be a pod